asciiracer/config.py

```diff
@@ -2,7 +2,6 @@
 # Definitions:
 # - [X]_STICKY_TIME: amount of time a key press of action [X]
 #                    sticks in the game
-GAME_SIZE = (29, 107)  # rows, cols
```

Comment:
I like the idea of having all non-state-related game variables in the config module. This would break it. Maybe keep that variable there and make it dynamic instead of hard-coded.